tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/IncClassEmitter.scala

```diff
@@ -0,0 +1,62 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js tools             **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2016, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+
+package org.scalajs.core.tools.linker.backend.emitter
+
+import scala.collection.mutable
+
+import org.scalajs.core.tools.sem._
+
+import org.scalajs.core.tools.linker.backend.OutputMode
+import org.scalajs.core.tools.linker.LinkingUnit
+
+/** Tracks incrementality breaking changes made during class emitting.
+ */
+final class IncClassEmitter(semantics: Semantics, outputMode: OutputMode) {
```

Comment:
This should be `private[emitter]`.

Of course I guess it can't, because of Rhino. So it should be `private[scalajs]` with a comment like the one in `ScalaJSClassEmitter`